assets/js/scenes/department/ProgrammesSection/index.html

```diff
@@ -18,29 +18,50 @@ <h3 class="Title Title--section">Spending plans by programme and sub-programme</
             <div class="ProgrammesSection-share u-marginTop10 u-marginBottom15">
                 <div data-component="Share" data-anchor="programmes" data-color="{{ color }}"></div>
             </div>
+{% if subprogramme_viz_data.get_openspending_api == None %}
 
-            <p class="js-tooltips">A department&#x27;s programmes are the activities that it spends money on during the
-                financial year. Different programmes have different budgets, depending on their objectives and available budgets. More detail on the programmes is available in the
-                department's {{ source_type }} documents.</p>
+<p class="js-tooltips">A department&#x27;s programmes are the activities that it spends money on during the
+    financial year. Different programmes have different budgets, depending on their objectives and available budgets. More detail on the programmes is available in the
+    department's {{ source_type }} documents.</p>
 
-            <ul class="LinksList">
-                {% include 'components/LinksList/item.html' with prefix="Source" text=source_text url=subprogramme_viz_data.get_dataset.get_url_path type="dataset" %}
+<ul class="LinksList">
+    
+    {% include 'components/LinksList/item.html' with text=guide_text url=guide type="guide" %}
 
-                {% include 'components/LinksList/item.html' with text="Download this data for analysis (CSV)" url=subprogramme_viz_data.get_detail_csv_url type="download" %}
+    {% include 'components/LinksList/item.html' with text=pdf_text url=pdf type="download" %}
 
-                {% include 'components/LinksList/item.html' with text=guide_text url=guide type="guide" %}
+    {% include 'components/LinksList/item.html' with text=excel_text url=excel type="download" %}
+</ul>
+</div>
 
-                {% include 'components/LinksList/item.html' with text=pdf_text url=pdf type="download" %}
+<div class="Card">           
+    <li> No data available </li>
+</div>
+           
+{% else %}
```

Comment:
instead of duplicating lots of markup, it's better to add an `if` around line 29:

```
{% include 'components/LinksList/item.html' with text="Download this data for analysis (CSV)" url=subprogramme_viz_data.get_detail_csv_url type="download" %}
```

as we have in assets/js/scenes/department/EconClassPackedCircles/econ-class-packed-circles.html

Reply:
done